resources/public/api/conf/6.0/loss_claims_delete.yaml

```diff
@@ -1,83 +1,84 @@
-  | NOT_FOUND                        | Simulates the scenario where no data is found. |
-  | OUTSIDE_AMENDMENT_WINDOW         | Simulates the scenario where request cannot be completed as it is outside the amendment window. |
-  | STATEFUL                         | Performs a stateful delete.                    |
-
 
-tags:
-  - Loss Claims
+  tags:
+    - Loss Claims
 
-parameters:
-  - $ref: './common/pathParameters.yaml#/components/parameters/nino'
-  - $ref: './common/pathParameters.yaml#/components/parameters/claimId'
-  - $ref: './common/headers.yaml#/components/parameters/acceptHeader'
-  - $ref: './common/headers.yaml#/components/parameters/authorizationWriteScopeHeader'
-  - $ref: './common/headers.yaml#/components/parameters/testScenarioHeader'
+  parameters:
+    - $ref: './common/pathParameters.yaml#/components/parameters/nino'
+    - $ref: './common/pathParameters.yaml#/components/parameters/claimId'
+    - $ref: './common/headers.yaml#/components/parameters/acceptHeader'
+    - $ref: './common/headers.yaml#/components/parameters/authorizationWriteScopeHeader'
+    - $ref: './common/headers.yaml#/components/parameters/testScenarioHeader'
 
-security:
-  - User-Restricted:
-      - write:self-assessment
+  security:
+    - User-Restricted:
+        - write:self-assessment
 
-responses:
-  "204":
-    description: Success (No Content)
-    headers:
-      X-CorrelationId:
-        $ref: './common/headers.yaml#/components/parameters/correlationId'
-    content: { }
+  responses:
+    "204":
+      description: Success (No Content)
+      headers:
+        X-CorrelationId:
+          $ref: './common/headers.yaml#/components/parameters/correlationId'
+      content: { }
 
 
-  "400":
-    description: Bad request
-    headers:
-      X-CorrelationId:
-        $ref: './common/headers.yaml#/components/parameters/correlationId'
-    content:
-      application/json:
-        schema:
-          $ref: './schemas/error_response.yaml'
-        examples:
-          FORMAT_NINO:
-            $ref: './common/errors.yaml#/components/examples/formatNino'
-          FORMAT_CLAIM_ID:
-            $ref: './common/errors.yaml#/components/examples/formatClaimId'
-          RULE_INCORRECT_GOV_TEST_SCENARIO:
-            $ref: './common/errors.yaml#/components/examples/ruleIncorrectGovTestScenario'
-          RULE_OUTSIDE_AMENDMENT_WINDOW:
-            $ref: './common/errors.yaml#/components/examples/ruleOutsideAmendmentWindow'
+    "400":
+      description: Bad request
+      headers:
+        X-CorrelationId:
+          $ref: './common/headers.yaml#/components/parameters/correlationId'
+      content:
+        application/json:
+          schema:
+            $ref: './schemas/error_response.yaml'
+          examples:
+            FORMAT_NINO:
+              $ref: './common/errors.yaml#/components/examples/formatNino'
+            FORMAT_CLAIM_ID:
+              $ref: './common/errors.yaml#/components/examples/formatClaimId'
```

Comment:
Below this, please add the errors for the newly introduced tax year. Can just grab it from [here](https://github.com/hmrc/individual-losses-api/blob/main/resources/public/api/conf/6.0/loss_claims_create.yaml#L87-L92), thanks.